.github/workflows/crds-gen.yml

```
@@ -37,8 +37,6 @@ jobs:
         with:
           go-version: "1.22"
 
-      - name: run crds-gen
-        run: GOPATH=$(go env GOPATH) make crds
+      - name: validate  crds-gen
```

Extra space intended?

```suggestion
      - name: validate crds-gen
```